api/_mixins/HTMLHyperlinkElementUtils__HTMLAreaElement.json

```
@@ -181,7 +181,7 @@
               "version_added": "≤12.1"
             },
             "safari": {
-              "version_added": "1"
+              "version_added": "3"
```

```suggestion
              "version_added": "1"
```

Source: https://trac.webkit.org/browser/webkit/trunk/WebCore/khtml/ecma/kjs_html.cpp?rev=4#L726